include/aspect/mesh_deformation/fastscape.h

```diff
@@ -0,0 +1,175 @@
+/*
+  Copyright (C) 2018 by the authors of the ASPECT code.
+  This file is part of ASPECT.
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_mesh_deformation_fast_scape_h
+#define _aspect_mesh_deformation_fast_scape_h
+
+#include <aspect/mesh_deformation/interface.h>
+
+/*
+ * Define FastScape functions as C functions. Must use exact same function/variable name
+ * and type as used in FastScape. All function names must be made lowercase, and an
+ * underscore added at the end. Types must be defined as pointers, and sent to
+ * fastscape as a reference.
+ */
+
+#ifdef __cplusplus
+extern"C" {
+#endif
+//Functions to initialize FastScape
```

Comment:
Do these functions have to be outside the FastScape plugin?

Reply:
It looks like they can be moved into either the aspect or mesh deformation namespace, but I get the error that they're out of scope if I put them into the FastScape class.